packages/@expo/cli/src/export/embed/index.ts

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env node
+import arg from 'arg';
+import chalk from 'chalk';
+import path from 'path';
+
+import { Command } from '../../../bin/cli';
+import { assertWithOptionsArgs, printHelp } from '../../utils/args';
+
+export const expoExportEmbed: Command = async (argv) => {
+  const rawArgsMap: arg.Spec = {
+    // Types
+    '--entry-file': String,
+    '--platform': String,
+    '--transformer': String,
+    '--bundle-output': String,
+    '--bundle-encoding': String,
+    '--max-workers': Number,
+    '--sourcemap-output': String,
+    '--sourcemap-sources-root': String,
+    '--assets-dest': String,
+    '--unstable-transform-profile': String,
+    '--config': String,
```

Comment:
Janic introduced a new `--asset-catalog-dest` option in https://github.com/react-native-community/cli/pull/1290. maybe we want to add this?